manifests/agent/service.pp

```diff
@@ -29,11 +31,23 @@
 
       $times = ip_to_cron($puppet::runinterval)
 
-      cron { 'puppet':
-        command => $command,
-        user    => root,
-        hour    => $times[0],
-        minute  => $times[1],
+      if $::osfamily == 'windows' {
+        scheduled_task { 'puppet':
+          ensure  => present,
+          enabled => true,
+          command => $command,
+          trigger => {
+            schedule   => daily,
+            start_time => $times[0],
```

Comment:
This document: https://docs.puppetlabs.com/puppet/3.6/reference/resources_scheduled_task_windows.html it says:
Puppet does not support “every X minutes” type triggers.

Also there is PUP-1291 that waits to be implemented.

I couldn't find how to do this type of scheduling with puppet on windows.

Also I noticed that $times[0] may be '*' which is not ok with windows. Should the schedule for windows be set to an hard coded hour, or removed?
The only thing that stops the module from running in windows is the cron type.